Code/Mantid/docs/source/algorithms/ConvertToReflectometryQ-v1.rst

```diff
@@ -23,6 +23,17 @@ workspace does not contain this value, or if you wish to override this
 value you can do so by providing your own *IncidentTheta* property and
 enabling *OverrideIncidentTheta*.
 
+The algorithm also has the option to create a table of 'DumpVertexes' that resemble
```

Comment:
Not quite true. The algorithm has the ability to provide additional debugging information that can be used to assemble patch plots of the results prior to 2D fractional rebinning.